taxtastic/taxonomy.py

```diff
@@ -86,8 +87,7 @@ def __init__(self, engine, NO_RANK='no_rank', schema=None):
         ranks = select([ranks_table.c.rank, ranks_table.c.height]).execute().fetchall()
         ranks = sorted(ranks, key=lambda x: int(x[1]))  # sort by height
         self.ranks = [r[0] for r in ranks]  # just the ordered ranks
-
-        self.NO_RANK = NO_RANK
+        self.unordered_ranks = set(self.ranks) & set(UNORDERED_RANKS)
```

Comment:
Global taxtastic.ncbi.UNORDERED_RANKS var difficult to avoid here without updating database schema to reflect this list